research/flamby/fed_heart_disease/perfcl/client.py

```diff
@@ -0,0 +1,177 @@
+import argparse
+import os
+from logging import INFO
+from pathlib import Path
+from typing import Optional, Sequence, Tuple
+
+import flwr as fl
+import torch
+import torch.nn as nn
+from flamby.datasets.fed_heart_disease import BATCH_SIZE, LR, NUM_CLIENTS, BaselineLoss
+from flwr.common.logger import log
+from flwr.common.typing import Config
+from torch.nn.modules.loss import _Loss
+from torch.optim import Optimizer
+from torch.utils.data import DataLoader
+
+from fl4health.checkpointing.checkpointer import (
+    BestMetricTorchCheckpointer,
+    LatestTorchCheckpointer,
+    TorchCheckpointer,
+)
+from fl4health.clients.fenda_client import FendaClient
+from fl4health.utils.losses import LossMeterType
+from fl4health.utils.metrics import Accuracy, Metric
+from fl4health.utils.random import set_all_random_seeds
+from research.flamby.fed_heart_disease.perfcl.perfcl_model import FedHeartDiseasePerFCLModel
+from research.flamby.flamby_data_utils import construct_fed_heard_disease_train_val_datasets
+
+
+class FedHeartDiseasePerFCLClient(FendaClient):
```

Comment:
Super minor: For acronyms, we've adopted the style of first letter uppercased and the rest lower. So PerFCL here would be `PerFcl`. This is similar for `FedHeartDiseasePerFCLModel` as well